### PR TITLE
Added 'Explore CC' Button on submit Page

### LINKED ISSUE #225: [Feature] "Add 'Explore CC' Button to Header of /submit Page
## Problem
The absence of the "Explore CC" button on the [submit](https://resources.creativecommons.org/submit/) page limits user exploration of Creative Commons resources. 

## Description

Currently, the "Explore CC" button is available on most pages of the Creative Commons website, facilitating easy discovery and navigation of CC-licensed content.Integrating this button into the page header will enhance usability and encourage users to engage with CC content while contributing.

## Implementation
Just need to add CC Global Header component on the submit file.

## Screenshots
Explore CC button missing from the top-right corner.
Screenshot 2024-03-20 at 09-18-20 Submit a resource Creative Commons

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by das123s**
## Fixes

Fixes #225 @praveen-rikhari

## Description
added the explore CC button on submit page

## Technical details
changes done on submission.html file

## Tests
1. Open submission page.
2. Explore CC button added to header of submit page.

## Screenshots

image
 
Image 2024-04-01 at 9 13 15 PM

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by possumbilities** [comment]
@das123s Hi! can you please fill out the PR description? If not I'll close this PR (you can always submit a new one with details later if that happens).

**Comment by das123s** [comment]
> @das123s Hi! can you please fill out the PR description? If not I'll close this PR (you can always submit a new one with details later if that happens).

**Comment by das123s** [comment]
> @das123s Hi! can you please fill out the PR description? If not I'll close this PR (you can always submit a new one with details later if that happens).

Yes, I have filled out the PR description. Could you please review it & if there is any changes required kindly let me know.

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -1,32 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (7.1.3.2)
```
Comment (possumbilities) [inline docs/Gemfile.lock:4]: There shouldn't be changes to this file in a PR here.

Comment (das123s) [reply docs/Gemfile.lock:4]: sure I will remove the changes of Gemfile.lock file

File: docs/Gemfile.lock
Comment (possumbilities) [inline docs/Gemfile.lock:1]: Hi! we don't want to delete this file, just revert the changes you accidentally made to it
